test/TesterInternal/ConfigTests.cs

```diff
@@ -833,7 +838,8 @@ public void ServerConfig_LoadFrom()
 
             Assert.Equal(filename, Path.GetFileName(config.SourceFile)); // OrleansConfiguration.SourceFile
         }
-
+#if !NETSTANDARD_TODO
+        //blocked by cannot update System.Xml.XmlDocument to 4.0.1 which is compatible with .net framework 4.6
```

Comment:
come over anytime . 
But the exception is 
Test Name:	UnitTests.ConfigTests.Config_StorageProviders_AzureTable_Default
Test FullName:	UnitTests.ConfigTests.Config_StorageProviders_AzureTable_Default
Test Source:	D:\microsoft\orleans-xiazen\test\TesterInternal\ConfigTests.cs : line 731
Test Outcome:	Failed
Test Duration:	0:00:00.284

Result StackTrace:	
at Orleans.Runtime.Configuration.ProviderConfiguration.LoadProviderConfigurations(XmlElement root, XmlNamespaceManager nsManager, IDictionary`2 alreadyLoaded, Action`1 add)
   at Orleans.Runtime.Configuration.ProviderCategoryConfiguration.Load(XmlElement child) in D:\microsoft\orleans-xiazen\src\Orleans\Configuration\ProviderConfiguration.cs:line 221
   at Orleans.Runtime.Configuration.GlobalConfiguration.Load(XmlElement root) in D:\microsoft\orleans-xiazen\src\Orleans\Configuration\GlobalConfiguration.cs:line 969
   at Orleans.Runtime.Configuration.ClusterConfiguration.LoadFromXml(XmlElement root) in D:\microsoft\orleans-xiazen\src\Orleans\Configuration\ClusterConfiguration.cs:line 99
   at Orleans.Runtime.Configuration.ClusterConfiguration.Load(TextReader input) in D:\microsoft\orleans-xiazen\src\Orleans\Configuration\ClusterConfiguration.cs:line 86
   at Orleans.Runtime.Configuration.ClusterConfiguration.LoadFromFile(String fileName) in D:\microsoft\orleans-xiazen\src\Orleans\Configuration\ClusterConfiguration.cs:line 204
   at UnitTests.ConfigTests.Config_StorageProviders_AzureTable_Default() in D:\microsoft\orleans-xiazen\test\TesterInternal\ConfigTests.cs:line 735
Result Message:	System.IO.FileLoadException : Could not load file or assembly 'System.Xml.XmlDocument, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a' or one of its dependencies. The located assembly's manifest definition does not match the assembly reference. (Exception from HRESULT: 0x80131040)